frontend/src/scenes/early-access-features/EarlyAccessFeatures.tsx

```diff
@@ -55,6 +65,17 @@ export function EarlyAccessFeatures(): JSX.Element {
                 }
                 delimited
             />
+            {showIntro && (
+                <ProductIntroduction
+                    productName="Early Access Features"
+                    productKey={ProductKey.EARLY_ACCESS_FEATURES}
+                    thingName="feature"
+                    description="Allow your users to individually enable or disable features that are in public beta."
+                    isEmpty={shouldShowEmptyState}
+                    docsURL="https://posthog.com/docs/feature-flags/early-access-feature-management"
+                    action={() => router.actions.push(urls.earlyAccessFeature('new'))}
+                />
+            )}
             <LemonTable
```

Comment:
In the other views (eg on feature flags or experiments) I only show the table if `sholdShowEmptyState === false`. It shows the empty state _and_ and empty table, which feels unnecessarily cluttered. But up to you, might not make sense for this view for some reason?

Reply:
done!